libs/arm64-v8a/gdb.setup

```diff
@@ -0,0 +1,2 @@
+set solib-search-path ./obj/local/arm64-v8a
+directory /home/radmin/Downloads/android-ndk-r15c/sysroot /home/radmin/Downloads/android-ndk-r15c/sources/android/support /home/radmin/Downloads/android-ndk-r15c/sources/android/support/include /home/radmin/Downloads/android-ndk-r15c/sources/cxx-stl/llvm-libc++ /home/radmin/Downloads/android-ndk-r15c/sources/cxx-stl/llvm-libc++abi /home/radmin/Documents/Public_Git_Projects/minicap/jni/libjpeg-turbo/include /home/radmin/Documents/Public_Git_Projects/minicap/jni/libjpeg-turbo/libjpeg-turbo-1.4.1/simd /home/radmin/Documents/Public_Git_Projects/minicap/jni/minicap-shared/aosp/include /home/radmin/Downloads/android-ndk-r15c/sources/cxx-stl/llvm-libc++/include /home/radmin/Downloads/android-ndk-r15c/sources/cxx-stl/llvm-libc++abi/include jni/libpng
```

Comment:
This file should not be committed. It contains paths specific to your machine.
The same applies to other gdb.setup files.

Reply:
Ok. let me remove the file

Reply:
FIle Removed